aiida/cmdline/commands/cmd_data/cmd_structure.py

```diff
@@ -206,7 +207,8 @@ def import_aiida_xyz(filename, vacuum_factor, vacuum_addition, pbc, label, group
 
     try:
         new_structure._parse_xyz(xyz_txt)  # pylint: disable=protected-access
-        new_structure._adjust_default_cell(  # pylint: disable=protected-access
+        new_structure = _adjust_default_cell(  # pylint: disable=protected-access
```

Comment:
Is there a specific reason you changed from a method to a function? Now it returns the structure, but it's not really a copy - it's actually changing it in place and then returning a reference to it, so I find it confusing (by just looking into the function signature, I would intuitively assume it does not change the first parameter but returns a copy).

Reply:
The only reason was that this is a function that is only used in a single place in aiida-core, and so this is a chance to reduce a little bit the enormous namespace of class methods.

>  Now it returns the structure, but it's not really a copy

This was already the case before.


In any case, it is not necessary for this PR, and I mentioned [above](https://github.com/aiidateam/aiida-core/pull/5341#issuecomment-1027130487) I'm fine to revert the change (which I will do).